class/person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age, parent_permission: true, name: 'Unknown')
+    super()
```

Comment:
- [ ] Kindly add a relationship between a person and a rental. Please implement what you have in the PR description. A person should have many rentals, so we need a method that will act as a has-many in a person's class. The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association, you can take a look at this article [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)